server/fractal-app.js

```diff
@@ -69,17 +69,20 @@ fractal.components.set('default.collator', function(markup, item) {
 });
 
 fractal.docs.set('path', dir('/views/docs'));
+fractal.docs.set('ext', 'njk');
+fractal.docs.engine(nunjucks);
+
 fractal.web.set('static.path', dir('./../dist'));
 fractal.web.set('builder.dest', dir('./../cardigan'));
 
 const cardiganTheme = mandelbrot({
   skin: 'navy',
-  styles: ['default', '/cardigan-theme/tweaks.css']
+  styles: ['default', 'dist-styles/styleguide.css']
```

Comment:
Where does this folder come from?

As [mentioned here](https://github.com/wellcometrust/wellcomecollection.org/pull/1063/files#r123951394) maybe we could just leave it in here for simplicities sake.